### PR TITLE
keycloak_user: avoid error when trying to delete a non-existing user

### PR DESCRIPTION
**Pull request by michield**
when force is set, but the user doesn't exist yet, this fails.

##### SUMMARY

Tiny fix to avoid trying to delete a user that may not exist

trivial:
  - keycloak_user: avoid deleting non-existent user

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

keycloak_user

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by michield** [comment]
> Thanks for your contribution! Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

Sorry, first time contributing here. Have added the trivial tag, hope I did it correctly.

**Comment by felixfontein** [comment]
@michield since this is a bugfix it should be `bugfixes:`. The `trivial:` section is reserved for things that shouldn't show up in the user-facing changelog.

Also in case you added a commit, you didn't push it :)

**Comment by michield** [comment]
> @michield since this is a bugfix it should be `bugfixes:`. The `trivial:` section is reserved for things that shouldn't show up in the user-facing changelog.
> 
> Also in case you added a commit, you didn't push it :)

Ah, hang on. I need to read better. I added it to the description of this PR. I'll make it bugfix and will add the commit to this PR.

File: changelogs/fragments/7696-avoid-attempt-to-delete-non-existing-user.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - keycloak_user: when force is set, but user does not exist, don't try to delete it
```
Comment (felixfontein) [inline changelogs/fragments/7696-avoid-attempt-to-delete-non-existing-user.yml:2]:
```suggestion
  - keycloak_user - when ``force`` is set, but user does not exist, do not try to delete it (https://github.com/ansible-collections/community.general/pull/7696).
```

**Comment by felixfontein** [comment]
If nobody objects in the next ~two weeks, I'll merge this.

**Comment by felixfontein** [comment]
@michield thanks for your contribution!

**Pull request merged**